nswag/Client.Class.ProcessResponse.liquid

```diff
@@ -26,7 +26,7 @@ var objectResponse_ = await ReadObjectResponseAsync<{{ response.Type }}>(respons
 {%         if response.IsNullable == false -%}
 if (objectResponse_.Object == null)
 {
-    throw new {{ ExceptionClass }}("Response was null which was not expected.", status_, objectResponse_.Text, headers_, null);
+    throw {{ ExceptionClass }}.Create(status_, headers_, null);
```

Comment:
This line handles the case when unable to deserialize a documented status code response. An `ApiException` with a `ProblemDetails` with minimal information will be thrown. See this commit to view what changes this line made to the generated code. 
https://github.com/Laserfiche/lf-repository-api-client-dotnet/pull/61/commits/44a382eb67730451bb3de7db985562f3e14775e3